examples/aws-ebs-csi-driver-self-managed/README.md

```diff
@@ -0,0 +1,40 @@
+## Requirements
```

Comment:
Lets not create a new example just for this - we can show and test its usage by adding it to https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/stateful